runtime/compiler/build/files/common.mk

```diff
@@ -392,7 +392,8 @@ JIT_PRODUCT_SOURCE_FILES+=\
     omr/compiler/runtime/OMRCodeCacheConfig.cpp \
     omr/compiler/runtime/OMRCodeCacheManager.cpp \
     omr/compiler/runtime/OMRCodeCacheMemorySegment.cpp \
-    omr/compiler/runtime/OMRRuntimeAssumptions.cpp
+    omr/compiler/runtime/OMRRuntimeAssumptions.cpp \
+    omr/compiler/runtime/OMRRSSReport.cpp
```

Comment:
I vaguely recall that there is another make file where you need to list the newly created files to be compiled

Reply:
Maybe in OMR? I can build OpenJ9.